observatory-platform/observatory/platform/docker/Dockerfile.package_install.jinja2

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Curtin University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Author: Tuan Chien
+
+USER ${INSTALL_USER}
+{% if package.type == 'editable' %}
+
+{% if install_deps %}
```

Comment:
Added a variable that lets you stop dependencies being installed to make installation of the package quicker.